docs/advanced-topics/tools-vs-workflows.rst

```diff
@@ -0,0 +1,117 @@
+Tools vs Workflows: Detailed Comparison
+=======================================
+
+.. tip::
+  Generally speaking, if you care more about the Docker image than the descriptor file, and are using CWL, you are dealing with a tool. If you care more about the descriptor file, you are dealing with a workflow.
+
+History
+-------
+When Dockstore was created, CWL was the first descriptor language we supported. CWL has a very clear distinction between a tool and a workflow. However, our definition for each does not completely align with the language's specification -- strictly speaking, a CWL CommandLineTool does not require a Docker image, but we require tools to be associated with a Docker image on Dockstore. Furthermore, while other descriptor languages do not have separate concepts for tools and workflows, we still maintain a distinction between a tool and a workflow for WDL for legacy reasons.
```

Comment:
> while other descriptor languages do not have separate concepts for tools and workflows, we still maintain a distinction between a tool and a workflow for WDL for legacy reasons

You shift from other descriptor tool languages to WDL, feels slightly off. And Galaxy may end up having tools... So how about, "... while WDL does not have separate concepts for tools and workflows, we still maintain a distinction between a tool and a workflow in WDL for legacy reasons."